Lombiq.HelpfulLibraries.LinqToDb/Readme.md

```diff
@@ -0,0 +1,56 @@
+# LinqToDb - Lombiq HelpfulLibraries for Orchard Core
+
+
+
+## About
+
+With the help of this module you can write LINQ expressions and run it with an `ISession` extension method to query from the DB instead of writing plain SQL queries.
+
+
+## Documentation
+
+Use the `LinqQueryAsync` ISession extension for running LINQ syntax based DB queries.
+```csharp 
+LinqQueryAsync<TResult>(this ISession session,Func<ITableAccessor, IQueryable> query)

```

Comment:
Ez inkább ne legyen itt, mert akkor itt is karban kell tartanunk. Elég a metódus neve, meg max kb. annyi, hogy "see its documentation inline".